### DISCUSSION TITLE
滴滴广告

### DISCUSSION THREAD

**Discussion by violesea** [blackmatrix7/ios_rule_script#354] [General]
我抓了几个可以的网址 但自己不会操作麻烦了。
request url：https://shop-gw.chengxinyouxuan.com/apolloConfig?city=&cityid=1&key=&sig=85062580a868c9ef893d181f7a678f336a9d1b6f&type=config_home_banner_v3%2Cconfig_seckill_entry%2Cconfig_new_scan_entry%2Cconfig_goods_mask%2Cconfig_receive_dispatch_detail%2Cconfig_search_entry%2Cconfig_slogan%2Cconfig_index_after_sales%2Cconfig_detail_after_sales%2Cconfig_sales_entry%2Cconfig_tuan_banner%2Cconfig_home_hots_cards%2Cconfig_classify_search%2Cconfig_home_hots_cards_v2%2Cconfig_manjian_banner%2Cconfig_manjian_rules%2Cconfig_manjian_landpage_banner%2Cconfig_kefu_flow%2Cconfig_show_apply_tuan%2Cconfig_goods_share_image%2Cconfig_731_downgrade%2Cconfig_aftersale_share_image%2Cconfig_generate_goods_share%2Cconfig_recommender_shopMP%2Cconfig_double11_activity%2Cconfig_share_cover%2Cconfig_user_logout%2Cconfig_poster_share_cover%2Cconfig_local_response_cache%2Cconfig_after_sale_switch%2Cconfig_webp_switch%2Cconfig_goodsland_url_switch%2Cconfig_spm%2Ctaxi_share_source_idslist%2Cis_new_pay_ok_page%2Cconfig_webp_gray_switch%2CisWXCloudEnable&u_source=0&u_uid=&uid=

{"data":{"pas_start_page":[{"T":"staticImage","activity_id":918849,"click_subtitle":"跳转详情页或第三方应用","click_tracks":["https://e.cn.miaozhen.com/r/k=2243951\u0026p=7tPUZ\u0026dx=__IPDX__\u0026rt=2\u0026pro=s\u0026ns=__IP__\u0026ni=__IESID__\u0026v=__LOC__\u0026xa=__ADPLATFORM__\u0026tr=c0a8006c60d0b09a6a106e8b100e0303\u0026ro=sm\u0026mo=1\u0026m0=__OPENUDID__\u0026m0a=__DUID__\u0026m1=__ANDROIDID1__\u0026m1a=__ANDROIDID__\u0026m2=__IMEI__\u0026m4=__AAID__\u0026m5=8caefc9c05d995b277dbc0ab18091d56\u0026m6=__MAC1__\u0026m6a=__MAC__\u0026m11=__OAID__\u0026m14=__CAID__\u0026m5a=__IDFV__\u0026mn=__ANAME__\u0026vo=3a244c4ed\u0026vr=2\u0026o=","https://adtrack.xiaojukeji.com/trackx?eid=6\u0026sg=53291c2ed5495ef6a42cefd16ed08e9d6f2d29ea3e66604283e9fa974e73babf\u0026res=18\u0026ht=1792\u0026openid=\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026cm=0\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026ant=16\u0026ori=1\u0026apv=6.2.2\u0026uid=281474976824227\u0026unt=787541\u0026lat=0.000000\u0026os=15.0\u0026wd=828\u0026cit=1\u0026lng=0.000000\u0026ct=0\u0026bt=1\u0026mt=0\u0026plt=1\u0026stt=0\u0026tkt=\u0026cost=0\u0026oid=\u0026idfa=0000000\u0026phb=iPhone12,1\u0026sld=1\u0026st=1624289435485\u0026chn=0\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026req=4445fcec-ea99-44a9-a3ed-ce677522fab8\u0026sch=223331\u0026act=918849\u0026bus=256"],"close_tracks":["https://adtrack.xiaojukeji.com/trackx?eid=7\u0026bus=256\u0026chn=0\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026req=4445fcec-ea99-44a9-a3ed-ce677522fab8\u0026sch=223331\u0026act=918849\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026sg=45f1e4846a364895614a8f616620896995bce1adaf5d72e23fc478ed3ae4bd79\u0026res=18\u0026ht=1792\u0026openid=\u0026cm=0\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026unt=787541\u0026ant=16\u0026ori=1\u0026apv=6.2.2\u0026uid=281474976824227\u0026lat=0.000000\u0026plt=1\u0026os=15.0\u0026wd=828\u0026cit=1\u0026lng=0.000000\u0026ct=0\u0026bt=1\u0026mt=0\u0026stt=0\u0026tkt=\u0026cost=0\u0026oid=\u0026st=1624289435485\u0026idfa=0000000\u0026phb=iPhone12,1\u0026sld=1"],"image":"https://img-ys011.didistatic.com/static/ad_oss/1623402260_pfa0q.jpg","imp_tracks":["https://g.cn.miaozhen.com/x/k=2243951\u0026p=7tPUZ\u0026dx=__IPDX__\u0026rt=2\u0026pro=s\u0026ns=__IP__\u0026ni=__IESID__\u0026v=__LOC__\u0026xa=__ADPLATFORM__\u0026tr=c0a8006c60d0b09a6a106e8b100e0303\u0026vg=__AUTOPLAY__\u0026nh=__AUTOREFRESH__\u0026mo=1\u0026m0=__OPENUDID__\u0026m0a=__DUID__\u0026m1=__ANDROIDID1__\u0026m1a=__ANDROIDID__\u0026m2=__IMEI__\u0026m4=__AAID__\u0026m5=8caefc9c05d995b277dbc0ab18091d56\u0026m6=__MAC1__\u0026m6a=__MAC__\u0026m11=__OAID__\u0026m14=__CAID__\u0026m5a=__IDFV__\u0026mn=__ANAME__\u0026o=","https://adtrack.xiaojukeji.com/trackx?eid=5\u0026tkt=\u0026cost=18\u0026oid=\u0026st=1624289435485\u0026idfa=0000000\u0026phb=iPhone12,1\u0026sld=1\u0026bus=256\u0026chn=0\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026req=4445fcec-ea99-44a9-a3ed-ce677522fab8\u0026sch=223331\u0026act=918849\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026sg=82a10294dfed6e4af45a2afe8c6b1ea4a479f5e06782bb87e457df38501a84c0\u0026res=18\u0026ht=1792\u0026openid=\u0026cm=0\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026unt=787541\u0026ant=16\u0026ori=1\u0026apv=6.2.2\u0026uid=281474976824227\u0026lat=0.000000\u0026plt=1\u0026os=15.0\u0026wd=828\u0026cit=1\u0026lng=0.000000\u0026ct=0\u0026bt=1\u0026mt=0\u0026stt=0"],"is_ad":1,"is_commercial_ad":true,"log_data":{"act_id":918849,"apolloGroupName":"","business_id":"0","city_id":1,"cost":18,"entrance_channel":"","entrance_yewuxian":0,"is_commercial_ad":true,"order_id":"","pvid":"e682c03e-a8f4-4efe-a611-920ba3efec0c","resource_id":"18","resource_name":"pas_start_page","schedule_id":"223331","slide_id":1,"trace_id":"c0a8006c60d0b09a6a106e8b100e0303","type":"engine","uid":281474976824227,"unit_id":"787541","work_flow_id":"eve_ads_default_20210427114416350-strategy"},"muilt_size":6,"priority":500,"resname":"pas_start_page","schedule_id":385345,"slide_id":1,"timesegs":[{"end_time":1624377599,"start_time":1624291200},{"end_time":1624550399,"start_time":1624464000}],"type":"staticImage","url":"https://id.vw.com.cn/id6campaign?mz_ca=2243951\u0026mz_sp=7tPUZ","useLogo":"1"},{"T":"staticImage","activity_id":911637,"click_subtitle":"跳转详情页或第三方应用","click_tracks":["https://adtrack.xiaojukeji.com/third?thirdtarget=https%3A%2F%2Fc.gridsumdissector.com%2Fr%2F%3Fgid%3Dgad_409_2dfl897j%26ck%3D61%26adk%3D1318%26os%3D1%26if%3D8caefc9c05d995b277dbc0ab18091d56%26oid%3D__OPENUDID__%26aid%3D__ANDROIDID__%26im%3D__IMEI__%26m%3D__MAC__%26ip%3D__IP__%26ts%3D__TS__%26did%3D__DUID__%26aaid%3D__AAID__%26uid%3D__UDID__%26odin%3D__ODIN__%26ua%3D__UA__%26lbs%3D47d19c2b5c2a7f0811c7fe0519008d14%26geo%3D__GEO__%26autorefresh%3D__AUTOREFRESH__%26caid%3D__CAID__","https://adtrack.xiaojukeji.com/trackx?eid=6\u0026cm=0\u0026idfa=0000000\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026oid=\u0026sg=ab9f85f2c4fc21c5a255d565b32ee103fbeddec1d4820e644290fc381226c9d3\u0026sch=221593\u0026os=15.0\u0026wd=828\u0026sld=6\u0026stt=0\u0026res=18\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026openid=\u0026lng=0.000000\u0026lat=0.000000\u0026cost=0\u0026req=d867ff79-3bfc-4ec7-9168-27de41eaf6a1\u0026act=911637\u0026ct=0\u0026ori=1\u0026plt=1\u0026phb=iPhone12,1\u0026chn=0\u0026apv=6.2.2\u0026cit=1\u0026unt=781331\u0026st=1624289435486\u0026bt=1\u0026mt=0\u0026tkt=\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026uid=281474976824227\u0026ant=16\u0026bus=256\u0026ht=1792"],"close_tracks":["https://adtrack.xiaojukeji.com/trackx?eid=7\u0026unt=781331\u0026st=1624289435486\u0026chn=0\u0026apv=6.2.2\u0026cit=1\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026uid=281474976824227\u0026ant=16\u0026bt=1\u0026mt=0\u0026tkt=\u0026bus=256\u0026ht=1792\u0026oid=\u0026sg=48d0e0f275ff748aafce61751e55007074cf45ffa1cbf3d587a17cab4cf9e6f7\u0026cm=0\u0026idfa=0000000\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026sld=6\u0026sch=221593\u0026os=15.0\u0026wd=828\u0026lng=0.000000\u0026lat=0.000000\u0026cost=0\u0026stt=0\u0026res=18\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026openid=\u0026ori=1\u0026plt=1\u0026phb=iPhone12,1\u0026req=d867ff79-3bfc-4ec7-9168-27de41eaf6a1\u0026act=911637\u0026ct=0"],"image":"https://img-ys011.didistatic.com/static/ad_oss/1622706631_7faiy.jpg","imp_tracks":["https://adtrack.xiaojukeji.com/trackx?eid=5\u0026bus=256\u0026ht=1792\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026oid=\u0026sg=e22c8fe99de48caecdb59538bb46325ebf24fbfd4fd06b4eb5bad3b681df040c\u0026cm=0\u0026idfa=0000000\u0026wd=828\u0026sld=6\u0026sch=221593\u0026os=15.0\u0026openid=\u0026lng=0.000000\u0026lat=0.000000\u0026cost=18\u0026stt=0\u0026res=18\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026ct=0\u0026ori=1\u0026plt=1\u0026phb=iPhone12,1\u0026req=d867ff79-3bfc-4ec7-9168-27de41eaf6a1\u0026act=911637\u0026cit=1\u0026unt=781331\u0026st=1624289435486\u0026chn=0\u0026apv=6.2.2\u0026tkt=\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026uid=281474976824227\u0026ant=16\u0026bt=1\u0026mt=0"],"is_ad":1,"is_commercial_ad":true,"log_data":{"act_id":911637,"apolloGroupName":"","business_id":"0","city_id":1,"cost":18,"entrance_channel":"","entrance_yewuxian":0,"is_commercial_ad":true,"order_id":"","pvid":"e682c03e-a8f4-4efe-a611-920ba3efec0c","resource_id":"18","resource_name":"pas_start_page","schedule_id":"221593","slide_id":6,"trace_id":"c0a8006c60d0b09a6a106e8b100e0303","type":"engine","uid":281474976824227,"unit_id":"781331","work_flow_id":"eve_ads_default_20210427114416350-strategy"},"muilt_size":6,"priority":500,"resname":"pas_start_page","schedule_id":383291,"slide_id":6,"timesegs":[{"end_time":1624377599,"start_time":1624204800}],"type":"staticImage","url":"https://hengchiad.evergrandeauto.com/20210322/common/index1.html?hmsr=didi0501","useLogo":"1"},{"T":"staticImage","activity_id":921717,"clickContent":"专享3折起","click_tracks":["https://adtrack.xiaojukeji.com/trackx?eid=6\u0026stt=0\u0026oid=\u0026res=18\u0026bus=256\u0026ht=1792\u0026openid=\u0026req=c419a6c3-313e-4013-8300-e20a4335304d\u0026uid=281474976824227\u0026unt=790217\u0026lng=0.000000\u0026sg=45e02f0717712dcb633609906418d3677f4665ec5c7899ff8c2d3f4ef46769d2\u0026sch=224081\u0026chn=0\u0026cm=0\u0026bt=1\u0026phb=iPhone12,1\u0026wd=828\u0026lat=0.000000\u0026ant=30\u0026ori=1\u0026mt=0\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026os=15.0\u0026ct=0\u0026tkt=\u0026sld=3\u0026cost=0\u0026plt=1\u0026apv=6.2.2\u0026cit=1\u0026idfa=0000000\u0026st=1624289435486\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026act=921717\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8="],"close_tracks":["https://adtrack.xiaojukeji.com/trackx?eid=7\u0026plt=1\u0026apv=6.2.2\u0026cit=1\u0026idfa=0000000\u0026st=1624289435486\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026act=921717\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026stt=0\u0026oid=\u0026res=18\u0026bus=256\u0026ht=1792\u0026openid=\u0026req=c419a6c3-313e-4013-8300-e20a4335304d\u0026uid=281474976824227\u0026unt=790217\u0026lng=0.000000\u0026sg=da7363945c56fc1a194635322976d6371131f82cfb9997beef30cd6d74280f2b\u0026sch=224081\u0026chn=0\u0026cm=0\u0026bt=1\u0026phb=iPhone12,1\u0026wd=828\u0026lat=0.000000\u0026ant=30\u0026ori=1\u0026mt=0\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026os=15.0\u0026ct=0\u0026tkt=\u0026sld=3\u0026cost=0"],"image":"https://img-ys011.didistatic.com/static/ad_oss/1623987781_2xvs1.jpg","imp_tracks":["https://adtrack.xiaojukeji.com/trackx?eid=5\u0026req=c419a6c3-313e-4013-8300-e20a4335304d\u0026uid=281474976824227\u0026unt=790217\u0026sch=224081\u0026chn=0\u0026cm=0\u0026bt=1\u0026lng=0.000000\u0026sg=60fdf1bae882089b0fa8e284e5006de808565dd662fd68ae881f5484bca3d0b0\u0026ori=1\u0026mt=0\u0026imei=10a84b418e17eda645ecf8d6f0b93f7a\u0026os=15.0\u0026phb=iPhone12,1\u0026wd=828\u0026lat=0.000000\u0026ant=30\u0026ct=0\u0026tkt=\u0026sld=3\u0026cost=18\u0026plt=1\u0026apv=6.2.2\u0026cit=1\u0026idfa=0000000\u0026st=1624289435486\u0026wkt=c0a8006c60d0b09a6a106e8b100e0303\u0026act=921717\u0026tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=\u0026res=18\u0026bus=256\u0026ht=1792\u0026openid=\u0026stt=0\u0026oid="],"is_ad":0,"is_commercial_ad":false,"log_data":{"act_id":921717,"apolloGroupName":"","business_id":"256","city_id":1,"cost":18,"entrance_channel":"","entrance_yewuxian":0,"is_commercial_ad":false,"order_id":"","pvid":"e682c03e-a8f4-4efe-a611-920ba3efec0c","resource_id":"18","resource_name":"pas_start_page","schedule_id":"224081","slide_id":3,"trace_id":"c0a8006c60d0b09a6a106e8b100e0303","type":"engine","uid":281474976824227,"unit_id":"790217","work_flow_id":"eve_ads_default_20210427114416350-strategy"},"muilt_size":6,"priority":500,"resname":"pas_start_page","schedule_id":386223,"slide_id":3,"timesegs":[{"end_time":1624463999,"start_time":1624204800}],"type":"staticImage","useLogo":"1"}]},"errmsg":"success","errno":0}

**Comment by violesea** [comment]
还有一个

url：https://res.xiaojukeji.com/resapi/activity/mget?sig=0c3c7501d6360e744d21bc1970707aa8f450eaa9

响应体：{
  "data": {
    "didipas_second_floor_home": [{
      "T": "dynamicIcon",
      "activity_id": 919689,
      "click_tracks": ["https://adtrack.xiaojukeji.com/trackx?eid=6&res=559&ct=0&wkt=c0a8006c60d0b09b6a106e8b102e0303&bus=256&os=15.0&st=1624289436028&lng=0.000000&cost=0&req=66a58a4a-9335-48d0-8686-716408d09034&phb=iPhone12,1&cit=1&ori=1&mt=0&sld=1&lat=0.000000&unt=788241&sg=99dd92b44785810a0a006f56c25cbd9bfa013a61123e917037e93165100c7a13&tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=&tkt=&idfa=0000000&plt=1&ht=1792&sch=223489&act=919689&cm=0&uid=281474976824227&ant=341&wd=828&stt=0&oid=&apv=6.2.2&openid=&chn=0&bt=1&imei=10a84b418e17eda645ecf8d6f0b93f7a"],
      "close_tracks": ["https://adtrack.xiaojukeji.com/trackx?eid=7&ori=1&mt=0&sld=1&lat=0.000000&unt=788241&sg=57555c0db784e317cacebd9d3acc1c2be01e74fa1af5221c90ab4ed18324798c&idfa=0000000&plt=1&ht=1792&sch=223489&act=919689&cm=0&tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=&tkt=&uid=281474976824227&ant=341&wd=828&stt=0&oid=&chn=0&bt=1&imei=10a84b418e17eda645ecf8d6f0b93f7a&apv=6.2.2&openid=&res=559&ct=0&bus=256&os=15.0&st=1624289436028&wkt=c0a8006c60d0b09b6a106e8b102e0303&req=66a58a4a-9335-48d0-8686-716408d09034&phb=iPhone12,1&cit=1&lng=0.000000&cost=0"],
      "icon": "https://img-ys011.didistatic.com/static/ad_oss/1623586393_m1goe.png",
      "imp_tracks": ["https://adtrack.xiaojukeji.com/trackx?eid=5&sld=1&ori=1&mt=0&sg=99cd0b8c37082cf5ac9f4bc51a89a89cdf4438e87f2a432192fb921f61fb8767&lat=0.000000&unt=788241&cm=0&tkn=zughJsnZ_p7v00gxnlaUYvNMS99IeJgdBE7-_L21SQskzD2OwkAMQOG7vNqKbI8nM3G7_d5hF8JPM0ggqoi7o0D_3rcxlKRMOinCMNKE4aSpqgqjkNbq4rVa0-jWhRGkzcXVWkQTRiX5-UX4I0H4J71btFja3D3cm3Ak93clNx635_2wkvoSTrvl4WEf60xiZTEtXms4wuVrXvf8HQAA__8=&tkt=&idfa=0000000&plt=1&ht=1792&sch=223489&act=919689&uid=281474976824227&ant=341&oid=&wd=828&stt=0&imei=10a84b418e17eda645ecf8d6f0b93f7a&apv=6.2.2&openid=&chn=0&bt=1&res=559&ct=0&st=1624289436028&wkt=c0a8006c60d0b09b6a106e8b102e0303&bus=256&os=15.0&cit=1&lng=0.000000&cost=0&req=66a58a4a-9335-48d0-8686-716408d09034&phb=iPhone12,1"],
      "is_ad": 0,
      "is_commercial_ad": false,
      "link": "https://v.didi.cn/622JKaK",
      "log_data": {
        "act_id": 919689,
        "apolloGroupName": "",
        "business_id": "256",
        "city_id": 1,
        "cost": 0,
        "entrance_channel": "",
        "entrance_yewuxian": 0,
        "is_commercial_ad": false,
        "order_id": "",
        "pvid": "b78fe9eb-ce8d-4039-98cd-04afd3c03880",
        "resource_id": "559",
        "resource_name": "didipas_second_floor_home",
        "schedule_id": "223489",
        "slide_id": 1,
        "trace_id": "c0a8006c60d0b09b6a106e8b102e0303",
        "type": "engine",
        "uid": 281474976824227,
        "unit_id": "788241",
        "work_flow_id": ""
      },
      "muilt_size": 1,
      "play_type": "0",
      "priority": 500,
      "resname": "didipas_second_floor_home",
      "schedule_id": 385521,
      "slide_id": 1,
      "timesegs": [{
        "end_time": 1624636799,
        "start_time": 1624204800
      }],
      "type": "dynamicIcon"
    }]
  },
  "errmsg": "success",
  "errno": 0
}

**Comment by blackmatrix7** [comment]
滴滴在我看来不算一个高频次长时间使用的App，所以暂时没有对其进行去广告的计划，抱歉帮不上你。
